### ISSUE TITLE
Installing Knative(docs/docs/snippets/prerequisites.md)

### ISSUE THREAD

**Issue by mukundjalan** [knative/docs#4356]
## Expected Behavior
Kubernetes cluster v1.19 can install Knative.

## Actual Behavior
But Kubernetes minimum version has changed to v1.20.0-0 and Install failed.

## Additional Info
**Additional context**
log from controller
```
{"severity":"EMERGENCY","timestamp":"2021-10-13T08:33:55.377066517Z","logger":"controller","caller":"sharedmain/main.go:273","message":"Version check failed","commit":"4a26f1d","knative.dev/pod":"controller-85d8685dcf-z44lb","error":"kubernetes version \"1.19.13-eks-8df270\" is not compatible, need at least \"1.20.0-0\" (this can be overridden with the env var \"KUBERNETES_MIN_VERSION\")","stacktrace":"knative.dev/pkg/injection/sharedmain.CheckK8sClientMinimumVersionOrDie\n\tknative.dev/pkg@v0.0.0-20210919202233-5ae482141474/injection/sharedmain/main.go:273\nknative.dev/pkg/injection/sharedmain.MainWithConfig\n\tknative.dev/pkg@v0.0.0-20210919202233-5ae482141474/injection/sharedmain/main.go:177\nknative.dev/pkg/injection/sharedmain.MainWithContext\n\tknative.dev/pkg@v0.0.0-20210919202233-5ae482141474/injection/sharedmain/main.go:131\nknative.dev/pkg/injection/sharedmain.Main\n\tknative.dev/pkg@v0.0.0-20210919202233-5ae482141474/injection/sharedmain/main.go:105\nmain.main\n\tknative.dev/serving/cmd/controller/main.go:45\nruntime.main\n\truntime/proc.go:225"}
```

**Install information:**
 - Platform : EKS
 - Knative Version: 0.26.0

**Comment by nak3** [comment]
Thank you for your report. https://github.com/knative/docs/pull/4355 and its backport will fix the issue.

**Comment by nak3** [comment]
> Kubernetes cluster v1.19 can install Knative.

Ah, sorry. Knative 0.26 requires Kubernetes v1.20. Is it possible to update your k8s cluster? 
Or you can workaround the min version via `https://github.com/knative/pkg/blob/main/version/version.go#L31` but it needs to be set every knative's system pod .

**Comment by snneji** [comment]
@mukundjalan The docs are fixed now (see #4355 and #4362), can I close this issue?
